### DISCUSSION TITLE
NASA試撞小行星任務 成功使其偏離自然軌道

### DISCUSSION THREAD

**Discussion by ABCDEFGHIJKLMNO12345678** [1101-GS4719-C/Python-Community-GS4719-C#761] [General]
在最近這幾年 科幻電影越來越普及 然而許多人們曾經覺得不可能的事 也一個一個化成了現實 而慧星撞地球的題材也不在畫下 「世界末日」（Armageddon）、「彗星撞地球」（Deep Impact）及「千萬別抬頭」（Don't Look Up）等科幻電影 也因這次NASA做的實驗 離我們生活越來越近

https://tw.news.yahoo.com/nasa%E8%A9%A6%E6%92%9E%E5%B0%8F%E8%A1%8C%E6%98%9F%E4%BB%BB%E5%8B%99-%E6%88%90%E5%8A%9F%E4%BD%BF%E5%85%B6%E5%81%8F%E9%9B%A2%E8%87%AA%E7%84%B6%E8%BB%8C%E9%81%93-011706861.html

**Comment by williamyen0623** [comment]
好像看雷射激光把彗星射爛

**Comment by ghost** [comment]
這次試驗讓我們未來面對直面而來的隕石時，有了新的一種對抗手段
